javawrapper/windows/packages.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Apache.QPID.Proton.AzureIot" version="0.9.0.1-preview-003" targetFramework="native" />
```

Comment:
Please remove dependency on review versions and replace with our latest Nuget packages.